src/Executor/ParallelExecutor.php

```diff
@@ -269,7 +268,9 @@ private function generateOptions(): string
             $inputs[] = Option::toString($this->input, $option);
         }
 
-        return implode(' ', $inputs);
+        return implode(' ', array_filter($inputs, function (string $item): bool {
```

Comment:
in this case `Lambda function` can be `static`. [for more info see](http://php.net/manual/en/functions.anonymous.php#functions.anonymous-functions.static)

Reply:
TIL!